### PR TITLE
balloon_hotplug: Add timeout value for unplug

### PR DESCRIPTION
**Pull request by huangyum**
Also update error message for unplug.

ID: 1714394, 1710669
Signed-off-by: Yumei Huang <yuhuang@redhat.com>

Depends on https://github.com/avocado-framework/avocado-vt/pull/2096

### CODE REVIEW

**Comment by huangyum** [comment]
Host_RHEL.m7.u7.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.balloon_hotplug.with_reboot: PASS (1008.06 s)

File: qemu/tests/balloon_hotplug.py [hunk 1]
```diff
@@ -116,9 +116,10 @@ def enable_balloon_service():
         error_context.context("Unplug balloon device for %d times" % (i+1),
                               logging.info)
 
-        out = vm.devices.simple_unplug(devs[0].get_aid(), vm.monitor)
+        out = vm.devices.simple_unplug(devs[0].get_aid(), vm.monitor,
+                                       timeout=90)
```
Comment (vivianQizhu) [inline qemu/tests/balloon_hotplug.py]: If RHEL linux guest take that long for unplug, should we be aware of it? If so I think better only give this long time out for windows.

Comment (huangyum) [reply qemu/tests/balloon_hotplug.py]: Updated, pls review, thanks.

**Comment by vivianQizhu** [comment]
@lijinlijin Request for your ack, thanks.

File: qemu/tests/balloon_hotplug.py [hunk 2]
```diff
@@ -116,9 +117,10 @@ def enable_balloon_service():
         error_context.context("Unplug balloon device for %d times" % (i+1),
                               logging.info)
 
-        out = vm.devices.simple_unplug(devs[0].get_aid(), vm.monitor)
+        out = vm.devices.simple_unplug(devs[0].get_aid(), vm.monitor,
+                                       timeout=unplug_timeout)
```
Comment (lijinlijin) [inline qemu/tests/balloon_hotplug.py]: Could you add the sleep step as we discussed in https://bugzilla.redhat.com/show_bug.cgi?id=1710669#c6?
Thanks

Comment (huangyum) [reply qemu/tests/balloon_hotplug.py]: @lijinlijin Updated, please help review, thanks.

**Comment by lijinlijin** [comment]
Acked-by: Li Jin lijin@redhat.com

**Comment by yanan-fu** [comment]
Thanks all!

**Pull request merged**